test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -14,6 +14,11 @@
 using NuGet.Packaging.PackageExtraction;
 using NuGet.Protocol;
 using Xunit;
+using NuGet.Common;
+using Newtonsoft.Json.Linq;
+using System.Text;
+using System.IO.Enumeration;
+using System.Runtime.CompilerServices;
```

Comment:
In Visual Studio navigate on your main menu to Tools | Options...
In the subsequent Options dialog navigate in the left-hand tree pane to Text Editor | C# | Advanced.
In the right-hand details pane, enable "Place 'System' directives first when sorting usings."
Click OK.
Right-click anywhere in this file and select "Remove and Sort Usings."